src/views/DisplayError.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div>
+    <v-alert type="error" color="pink darken-4">
+      <v-row align="center">
+        <v-col class="grow">{{ this.errorMessage }}</v-col>
+        <v-col class="shrink"> </v-col>
+      </v-row>
+    </v-alert>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "DisplayError",
+  computed: {
+    errorMessage: function() {
+      let message = "";
+      switch (this.$route.params.id) {
+        case 1:
+          message =
+            "Your API key is absent. To run the app, please make a file named  '.env.local'. Please visit https://developers.zomato.com/api#headline2 and request your API key. Then write in env.local: VUE_APP_API_KEY=your API KEY. Save it. Run in console npm run serve. For more information, read readme file.";
```

Comment:
"Please" isn't needed, as it's not really a request. It's instruction. Other than that, "Then write in env.local" starts with a capital letter after comma and lacks the dot prefix in filename.

Also: it would be even better if there was a link instead of an URL (and also a link to the readme, while we're at that).

Reply:
> key. Then 

Here? There is no comma here?